symforce/examples/bundle_adjustment_in_the_large/bundle_adjustment_in_the_large.cc

```diff
@@ -0,0 +1,112 @@
+/* ----------------------------------------------------------------------------
+ * SymForce - Copyright 2022, Skydio, Inc.
+ * This source code is under the Apache 2.0 license found in the LICENSE file.
+ * ---------------------------------------------------------------------------- */
+
+#include <fstream>
+
+#include <sym/pose3.h>
+#include <symforce/opt/factor.h>
+#include <symforce/opt/internal/logging_configure.h>
+#include <symforce/opt/optimizer.h>
+#include <symforce/opt/values.h>
+
+#include "./gen/keys.h"
+#include "./gen/snavely_reprojection_factor.h"
+
+using namespace sym::Keys;
+
+sym::Factord MakeFactor(int camera, int point, int pixel) {
+  return sym::Factord::Hessian(sym::SnavelyReprojectionFactor<double>,
+                               {
+                                   sym::Key::WithSuper(CAM_T_WORLD, camera),
```

Comment:
Curious if there's a reason this is with a superscript vs subscript via `sym::Key(CAM_T_WORLD, camera)` or `{CAM_T_WORLD, camera}`

Reply:
`values_codegen.generate_values_keys` will generate keys with subscripts, which is necessary if you have more than 52 keys or so.  In this case all of our generated keys happen to not have subscripts, but the general way to use this mechanism (at least the way it's currently designed) is for the subscripts to be reserved for codegen time and superscripts for runtime